packages/react-ui/internal/themes/DefaultTheme.ts

```diff
@@ -847,6 +850,8 @@ export class DefaultTheme {
   }
   public static menuItemCommentColor = '#a0a0a0';
   public static menuItemCommentColorHover = '#fff';
+
+  public static mobileMenuItemPadding = '12px 16px';
```

Comment:
Не стоит ли в таких переменных `modile` добавлять в конце, как модификатор? Вначале идет имя компонента, будет путаница с `mobilePopup...`. 
```suggestion
  public static menuItemPaddingMobile = '12px 16px';
```